assets/prototype/application/services/config/useConfig.ts

```diff
@@ -0,0 +1,12 @@
+import { useContext } from 'react';
+import { ConfigContext } from '../context/ConfigProvider';
+
+type useConfigReturnType = {
```

Comment:
Open for a better naming here.

Reply:
I think we should avoid using `use` at the beginning of TS types to avoid confusion with React Hooks

Reply:
Updated, thx.